### ISSUE TITLE
Add multilingual support for cinemeta, just like tmdb addon

### ISSUE THREAD

**Issue by shanyan-wcx** [Stremio/stremio-features#236]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by BlackPauloBR** [comment]
The movie/series synopsis is in a different UI language.  Ex.  I always get frustrated when I open a movie and I can't read the synopsis because my native language is different from English, all synopses of any content are always in English, and also when watching the trailer of a movie / series, I go through the same frustration , there is an extension of The Movie Database, which helps with this difficulty, as it allows you to install it in my language (Portuguese-Brazil), to access the synopsis, trailers, movie cover, and loading image translated into my language, the trailer already has subtitles in Portuguese, the presentation cover of the films is with the title that was released in my country (Brazil), the torrent loading image is also an image with sayings translated into Portuguese, everything is beautiful, perfect, more it is not practical to access content via extension in this way, it is necessary to access the search filter, and select the content of that extension.  because it is not practical, and brings duplication of content, as the content of the extension (translated) and the standard content of the Cinemeta system appears, I am here asking for accessibility in any country to be achieved.  This could be a standard functionality of stremio, change the language of Cinemeta.  I believe that this solution in the official Cinemeta extension would be very welcome in the user experience for people who speak a language other than English.


 The Cinemeta language could be associated with the user interface language.

  It could have in the stremio application settings, just below the interface language setting, an option to select the language of Cinemeta, or it could automatically follow the language configured in the interface.
 If the addition of this feature is too complex, they could give the option to disable Cinemeta, so that duplicate content does not appear, since The Movie Database extension is applying this necessary translation and displaying the exploration content.

**Comment by soyelmismo** [comment]
This is a very esthetic feature that should be priorized...

**Comment by lmichaudel** [comment]
+1 should be prioritized, it's an important QoL change, especially for family members who don't speak english..

**Comment by lenylvt** [comment]
Any Update ?

**Comment by lmichaudel** [comment]
This has been asked for years and Stremio have never provided us any ETA for this.
Moreover stremio add-ons and cinemata are NOT open source so the community can not implement it.

This is just ridiculous for an "open source" project.

**Comment by T0unet** [comment]
Is it possible to prioritise multilingual support in cinemata? as was done for the stremio ui.

**Comment by ruimendes11** [comment]
What's the best workaround for this please?

**Comment by soyelmismo** [comment]
> the best workaround

at the moment, tmdb addon

**Comment by Casadomoticap** [comment]
This would be amazing for this new year 2025. This is a basic thing to do for improve Stremio. It has been asked for years... Please developers, prioritize this :(

**Comment by vhrita** [comment]
Many addons are better than Cinemata, but we can't disable it. It keeps appearing at the top of search results. Please consider giving us the option to deactivate Cinemata from search results or allow us to prioritize another addon to appear at the top instead. It's not the issue solution, but will help us a lot, because we will be able to choose the addon that shows results in our native language.

**Comment by btd1337** [comment]
Developers, this feature is the main obstacle preventing Stremio from becoming the leading streaming player.

Not having movie titles and descriptions translated into the users' native language is a fundamental mistake. Even amateur developers creating basic apps make this essential feature a priority.

Please reconsider working on this feature.

**Comment by wrs92** [comment]
any update?

**Comment by bartned** [comment]
+1

**Comment by 50P15** [comment]
+1

**Comment by marcosschroer** [comment]
Exactly, Stremio could be number 1 by now... Specially considering that the famous" My Family Cinema" here in Brazil has been turned off, more than 6 million users are migrating to different platforms.
It requires too much work for someone to Install the add-ons and set-up everything to have the descriptions and titles in portuguese, and it's not even that perfect.

**Comment by marcoshimselfxd** [comment]
> What's the best workaround for this please?

The Movie Database add-on
or AIOMetadata with TMDB configured

**Comment by AngeloFilaseta** [comment]
This feature would be game changing.
Using other add-ons usually leads to instabilities in usage. 

I would suggest prioritizing this feature, the internet is flooded with workarounds that just partially work  and are time wasting.

**Comment by draxx31** [comment]
Hello, is this issue taken into consideration by streamio team?

**Comment by rodhfr** [comment]
would be indeed great addition

**Comment by kKaskak** [comment]
Cinemeta configuration feature is available now for all supporters which enables different langs both for posters and for the titles as well so you can have full catalogs in your preffered lang

https://stremio.com/donate
